dll/win32/browseui/addressband.cpp

```diff
@@ -259,24 +259,52 @@ HRESULT STDMETHODCALLTYPE CAddressBand::HasFocusIO()
     return S_FALSE;
 }
 
-HRESULT STDMETHODCALLTYPE CAddressBand::TranslateAcceleratorIO(LPMSG lpMsg)
+WCHAR CAddressBand::GetAddressBarFocusKey()
 {
-    if (lpMsg->hwnd == fEditControl)
+    static WCHAR s_szKey[2] = L"";
+    if (s_szKey[0] == UNICODE_NULL)
     {
-        switch (lpMsg->message)
+        if (!LoadStringW(_AtlBaseModule.GetResourceInstance(), IDS_ADDRESSBARFOCUSKEY,
```

Comment:
Are you sure that this should be special cased like this,
and not just be a 'regular' accelerator?

Reply:
![image](https://user-images.githubusercontent.com/2107452/217509349-81a6cd69-bd7e-4ded-9bd6-2377ec3f695b.png)
`WM_SYSKEYDOWN` hits `LoadStringW` and there is `"D"` resource string in `shell32`.

Reply:
The normal accelerator generates `WM_COMMAND` message.

Reply:
![image](https://user-images.githubusercontent.com/2107452/217511504-13800bfb-77cc-48df-95bb-97274dcb07cb.png)
`browseui` doesn't have `RT_ACCELERATOR`.

![image](https://user-images.githubusercontent.com/2107452/217511767-2f2360b9-40ba-436e-a4d7-425b22bc6c1a.png)
`shell32` doesn't have `Alt+D` accelerator.

Reply:
We don't have `BROWSEUI!MLLoadStringW`.